src/shared/containers/MapContainer.js

```diff
@@ -6,56 +6,54 @@ import { getMapMarkers } from '../actions/map';
 import MapBlock from '../components/MapBlock.js';
 
 
-class MapContainer extends Component {		
-
-	constructor(props) {
-		super(props);
-		this.shouldComponentUpdate = shouldPureComponentUpdate;
-	}
-
-	componentDidMount() {
-
-		this.props.getMapMarkers().then(
-			() => {
-				console.log("dd:"+this.props.status + "/"+JSON.stringify(this.props.markers.data));
-			}
-		);
-	}
-	
-
-	render() {
-		return (
-			<section>
-				<h2>test map</h2>
-				<MapBlock markers={this.props.markers.data} />				
-			</section>
-		);
-	}
+class MapContainer extends Component {
+
+  constructor(props) {
+    super(props);
+    this.shouldComponentUpdate = shouldPureComponentUpdate;
+  }
+
+  componentDidMount() {
+    this.props.getMapMarkers().then(
+      () => {
+        console.log('dd:' + this.props.status + '/' + JSON.stringify(this.props.markers.data));
+      }
+    );
+  }
+
+  render() {
+    return (
+      <section>
+        <h2>test map</h2>
+        <MapBlock markers={this.props.markers.data} />
+      </section>
+    );
+  }
 }
 
 MapContainer.propTypes = {
-	status: PropTypes.string,
-	markers: PropTypes.object,
-	getMapMarkers: PropTypes.func
+  status: PropTypes.string,
+  markers: PropTypes.object,
+  getMapMarkers: PropTypes.func
 };
 
 MapContainer.defaultProps = {
 
 };
 
 const mapStateToProps = (state) => {
-	return {
-		status: state.map.get.status,
-		markers: state.map.get.markers
-	};
+  return {
+    status: state.map.get.status,
+    markers: state.map.get.markers
+  };
 };
 
 const mapDispatchToProps = (dispatch) => {
```

Comment:
You don't need return statement for a single return block like below:
`
const mapDispatchToProps = dispatch => {
   getMapMarkers: () =>  dispatch(getMapMarkers())
}
`